Jenkinsfile

```diff
@@ -101,8 +101,12 @@ helpers.rootLinuxNode(env, {
               }
               checkDiffs(['./go/', './protocol/'], 'Please run \\"make\\" inside the client/protocol directory.')
             }
+
             parallel (
               test_linux: {
+                // Make sure vendored stellar external packages haven't changed
+                sh 'make shavendorstellar'
+
                 def packagesToTest = [:]
                 if (hasGoChanges) {
```

Comment:
I'd put it right here.

Reply:
how do I ensure cwd is correct?  (it currently isn't)

Reply:
Oops sorry about the delay. Looks like you got it.